### ISSUE TITLE
Number of missed blocks

### ISSUE THREAD

**Issue by themantre** [pactus-project/pactus#624]
## Describe the problem you'd like to have solved

There may be situations where a validator, acting as a proposer, misses a block. 
In these cases, we should keep track of the number of missed blocks. 
This metric will aid users in diagnosing issues with their validator and fix the problem, if any exist. 
A block can be missed if a validator is faulty (for example, if the system timestamp is [significantly off](https://github.com/pactus-project/pactus/discussions/611)) or if it has a very poor internet connection.

## Describe the ideal solution

Once a proposer missed a block, we should increase a counter inside state to let user know about the missed blocks number.

## Alternatives and current work-around

There is no clear workaround, unless user manually scan all the blocks.
Which is not straight forward.

**Comment by kehiy** [comment]
@themantre Currently in Pactus nodes with the wrong time config, they can't connect to the network. the PIP-19 score will help them find out about poor internet connection.

**Comment by b00f** [comment]
No, still we need to report it to user that how many blocks they have missed
